release-content/0.14/release-notes/13317_12502_Remove_limit_on_RenderLayers.md

```diff
@@ -1,4 +1,31 @@
 <!-- #12502 Remove limit on RenderLayers. -->
 <!-- https://github.com/bevyengine/bevy/pull/13317 -->
 
-<!-- TODO -->
+In Bevy you are able to assign entities spawned to different layers.
+
+You can make 3D models, meshes, lights, etc. all have their own if you wish.
+
+Then you can tell cameras which layers to render, and only objects which are part of those layers get rendered to that camera's target.
+
+```rust
+// This camera shows everything by default- membership in layer 0 is implicit
+commands.spawn(Camera3dBundle::default());
+
+const DEBUG_EXTRAS: usize = 100;
```

Comment:
This probably doesn't *actually* matter in practice, but layer 100 will require 2 bitmasks, which maybe isn't great to model for users.

Reply:
I'm going to update this PR later today when I get home- since for most users the change is simply _use as many render layers as you want_ I'm thinking of just removing code examples from this PR and just having a few sentences (including what you suggested).

Perhaps `RenderLayerManager` etc. could grow into a proper example in Bevy itself in the future, afaik we have no dedicated example for render layers.

Reply:
Great! Yeah, I think a. example isn't really necessary here. :)